Examples/test-suite/fortran_shared_ptr.i

```diff
@@ -0,0 +1,25 @@
+/* File: fortran_shared_ptr.i */
```

Comment:
Can you remove this test and use the numerous and more comprehensive shared_ptr tests instead? fortran_shared_ptr doesn't seem to do anything that the current tests already do. The tests beginning with a language name ought to be testing language specific features.

Reply:
This file is for a minimal test (see `fortran_shared_ptr_runme.f90`) for checking that the Fortran assignment `generic(=)` is properly releasing and retaining the shared pointers. The gymnastics required for a workable assignment operator in Fortran require thorough testing, and a simple use case like this is beneficial IMHO.

Reply:
OK I understand now -- I was looking for `boost_shared_ptr` or `shared_ptr` but didn't see them (derp). I've removed the `fortran_shared_ptr` test and replaced with an `li_boost_shared_ptr_runme.f90` test.